mm/mm_heap/mm_free.c

```diff
@@ -42,8 +42,31 @@
 #include <assert.h>
 #include <debug.h>
 
+#include <nuttx/arch.h>
+#include <nuttx/irq.h>
 #include <nuttx/mm/mm.h>
 
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+#ifdef __KERNEL__
+static void mm_add_delaylist(FAR struct mm_heap_s *heap, FAR void *mem)
+{
+  FAR struct mm_delaynode_s *new = mem;
```

Comment:
Change new to tmp like mm_free_delaylist

Reply:
Good catch.  You have very good eyes, Xiang.  I have already merged the change, but new will cause problems for some tools.